Files/MultilingualResources/Files.fr-FR.xlf

```diff
@@ -3561,133 +3561,133 @@ Nous utilisons l'App Center pour suivre l'utilisation de l'application, trouver
         </trans-unit>
         <trans-unit id="OpenLogLocation" translate="yes" xml:space="preserve">
           <source>Open log location</source>
-          <target state="new">Open log location</target>
+          <target state="new">Ouvrir l'emplacement des fichiers journaux</target>
```

Comment:
Can you update the state to `translated`?

Reply:
Done!